ssclj/jar/src/main/clojure/com/sixsq/slipstream/ssclj/db/impl.clj

```diff
@@ -0,0 +1,26 @@
+(ns com.sixsq.slipstream.ssclj.db.impl
+  (:require
+    [com.sixsq.slipstream.ssclj.db.binding :as p]
+    [clojure.tools.logging :as log]))
+
+(def ^:dynamic *impl*)
+
+(defn set-impl!
+  [impl]
+  (alter-var-root #'*impl* (constantly impl)))
```

Comment:
@loomis 
I don't have the full overview of this code and therefore I wonder why a `binding` form or a `(def impl (delay...` form are not used instead of changing the value of `*impl*` with `alter-var-root`, which will affect the value on all threads.